test/spec/core/jhipster/reserved_keywords_test.js

```diff
@@ -38,6 +38,7 @@ describe('ReservedKeywords', () => {
         expect(isReserved('ADD', 'CASSANDRA')).to.be.true;
         expect(isReserved('ACTIVATE', 'ORACLE')).to.be.true;
         expect(isReserved('DOCUMENT', 'MONGODB')).to.be.true;
+        expect(isReserved('DOCUMENT', 'COUCHBASE')).to.be.false;
```

Comment:
You should remove this line.

Reply:
I'll modify to check a real reserved word.